public/components/common/search/search.tsx

```diff
@@ -271,16 +272,16 @@ export const Search = (props: any) => {
 
   useEffect(() => {
     // set index and olly query assistant question if changed elsewhere
-    if (!queryRedux.ollyQueryAssistant) return;
-    if (queryRedux.index.length > 0) {
+    if (!queryRedux?.ollyQueryAssistant) return;
+    if (queryRedux?.index.length > 0) {
```

Comment:
i thought there are default values for these?

Reply:
There are default values, I'm changing this back and also modifying the test I aimed to pass originally.